spacy/tests/doc/test_span.py

```diff
@@ -434,3 +439,59 @@ def test_span_with_vectors(doc):
     # single-token span with vector
     assert_array_equal(ops.to_numpy(doc[10:11].vector), [-1, -1, -1])
     doc.vocab.vectors = prev_vectors
+
+def test_span_sents(doc, doc_not_parsed) :
+    # Entire doc
+    span = Span(doc, 0, len(doc))
+    sentences = list(span.sents)
+    assert len(sentences) == 3
```

Comment:
To shorten the code, I'd remove the var from all of these checks:
```suggestion
    assert len(list(span.sents)) == 3
```